frameworks/helloworld/src/test/java/com/mesosphere/sdk/helloworld/scheduler/ServiceTest.java

```diff
@@ -344,6 +344,150 @@ private void testWorldDecommission(boolean useDefaultExecutor) throws Exception
         runner.run(ticks);
     }
 
+
+    @Test
```

Comment:
It looks like the main differences between these tests are:
- The readiness check exit code
- The resulting status
- Whether to use custom or default executor

Given that, could break the test out into a separate parameterized function and have these three call into that function?